recipes/pyinaturalist-convert/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "pyinaturalist-convert" %}
+{% set version = "0.2.0" %}
+{% set hash_val = "7c02565c8100d9bc05b63872858ec7c13f88f441c3d64c702e29e6991efc3cbd" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ hash_val }}
```

Comment:
These days we usually just put the hash directly here rather than templating it.